packages/components/pie-modal/src/index.ts

```diff
@@ -165,34 +179,24 @@ export class PieModal extends RtlMixin(LitElement) {
             event.clientX < left ||
             event.clientX > right;
 
-        if (isClickOutsideDialog) {
+        if (isClickOutsideDialog && this.isDismissible) {
```

Comment:
We could probably move the `isDismissible` check right to the beginning of the function to avoid doing all the calculations

Reply:
Yeah agree something i thought about when i was looking at this yesterday but forgot to actually do it 👍